tests/ledctl/parameters_test.py

```diff
@@ -50,3 +50,49 @@ def test_ibpi_parameters_are_valid_short_test_flag(ledctl_binary, valid_ibpi_com
     cmd = LedctlCmd(ledctl_binary)
     cmd.is_test_flag_enabled()
     cmd.run_ledctl_cmd(valid_ibpi_commands.split())
+
+@pytest.mark.parametrize("valid_warning_level_commands", [
```

Comment:
in all test you did following:
```
"--list-controllers" + log_level +T
"normal=/dev/nvme0n1 --llisted-only" + log_level -T
"-P -n vmd" +level -T
"normal=/dev/nvme0n1 --listed-only" +level -T
```
All what should be parametrized are levels you are using, not whole cmdline.
Mode should not matter too, you can test one mode, it will be fine (if someone would remove log_level from comon_opts define that it will be discovered fast).